src/main/java/io/cdap/e2e/pages/locators/CdfStudioLocators.java

```diff
@@ -75,7 +75,10 @@
   @FindBy(how = How.XPATH, using = "//*[@data-cy='preview-configure-run-btn']")
   public static WebElement configRun;
 
-}
-
+  @FindBy(how = How.XPATH, using = "//*[contains(@data-cy,'plugin-node-BigQueryMultiTable')]")
+  public static WebElement bigQueryMultiTable;
```

Comment:
It does not seem reasonable to put a separate element for each plugin we plan to support. We should be able to get those using plugin name